src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -4662,6 +4666,8 @@ public void testReadonlyRootfs() throws Exception {
   public void testStorageOpt() throws Exception {
     requireDockerApiVersionAtLeast("1.24", "StorageOpt");
     requireStorageDriverNotAufs();
+    // Doesn't work on Travis because storage driver doesn't have pquota mount option enabled.
+    assumeFalse(dockerApiVersionEquals("1.32") && TRAVIS);
```

Comment:
A boolean OR seems more logical here, to me. If either are true, we want to skip the test.

Reply:
Oh it’s meant to skip if both on Travis and that version.

Reply:
I've clarified this comment.

Reply:
_Only_ this version is broken? Is that documented on https://travis-ci.org/ anywhere?

Reply:
So this version requires pquota to be set regardless of Travis or not. We
can set it on a host. But to be activated I think it requires a reboot of
the host. This obviously doesn’t work on Travis. That’s why the cobination
of the version and Travis don’t work.